site/configure.xml

```diff
@@ -497,71 +702,87 @@ CONFIG_FILE=/etc/rabbitmq/testdir/bunnies</pre>
           </tr>
           <tr>
             <td>
-              <code>tcp_listeners</code>
+              <code>listeners</code>
             </td>
             <td>
-              List of ports on which to listen for AMQP connections
-              (without SSL). Can contain integers (meaning "listen on
-              all interfaces") or tuples such as <code>{"127.0.0.1",
-              5672}</code> to listen on one or more interfaces.
+              Ports or hostname/pair on which to listen for AMQP connections
+              (without TLS). See the <a href="/networking.html">Networking guide</a> for more
+              details and examples.
 
-              <p>Default: <code>[5672]</code></p>
+              <p>Default: <code>listeners.tcp.default = 5672</code></p>
```

Comment:
1. TLS is over TCP but there is no word for "non-TLS" ("plain" is not widely used).
2. It's a name that can be anything.
3. Management UI probably nearly always has a single listener configured, so this is in part by design.
4. We could replace `none` with `false` but `none` means something else: it means "no options"
5. We discussed this before. If we change this to `ssl_options` it will confuse those who are used to the classic format, which is the vast majority of people. The best I can suggest is that we support both in the management plugin.